src/collections/db.ts

```diff
@@ -87,6 +87,9 @@ export class Db {
    * @returns Promise
    */
   async dropDatabase() {
+    if(this.rootHttpClient.isAstra){
+      throw new StargateAstraError('Cannot drop database in Astra. Please use the Astra UI to drop the database.');
```

Comment:
@vkarpov15 Instead of this, shall we get all collections, loop through and drop the collections in the case of Astra (to cover the use case of dropDatabase in any mongoose app tests)?

Reply:
But this is not a blocker for this PR. We can take this up separately if we would like to discuss further on this (i.e. dropDatabase in Astra)

Reply:
We should discuss dropDatabase() in Astra further, but I'm fine with looping through all collections here.